dist/worker-form.tpl.html.js

```diff
@@ -69,7 +69,7 @@ ngModule.run(['$templateCache', function ($templateCache) {
     '\n' +
     '  <md-input-container class="md-block">\n' +
     '    <label for="workername">Phone number</label>\n' +
-    '    <input type="number" id="phonenumber" name="phonenumber" ng-model="ctrl.model.phone" pattern="([0-9]{7,15})" required>\n' +
+    '    <input type="text" id="phonenumber" name="phonenumber" ng-model="ctrl.model.phone" pattern="([0-9]{7,15})" required>\n' +
```

Comment:
[Hint] There is input type dedicated for telephone supported in any major browser:
 `<input type="tel">` 
https://www.w3schools.com/html/html_form_input_types.asp